src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -125,4 +125,24 @@ public String getNewQuestion(Entity game_entity, DatastoreService datastore) {
         return (update_game_entity.getProperty("quizQuestion")).toString();
     }
 
+    public Boolean giveUserPoints(Boolean userQuizStatus, Entity currentUser, DatastoreService datastore) {
```

Comment:
maybe give a line or two of documentation for this function

Reply:
It would help to maybe be a little more specific here as well. "Gives the user points" how many points? How does it give them to the user? What is a "user"?

Something like "Adds 20 points to the given User Entity for completing a quiz."